docs/contributing/add_new_resource.md

```diff
@@ -18,10 +18,12 @@ The first step in determining if a GCP resource is supported is to figure out th
 
 A resource is "supported" by terraform-validator if it has an entry in [mappers.go](https://github.com/GoogleCloudPlatform/terraform-google-conversion/blob/master/google/mappers.go). For example, you could search mappers.go for [`google_compute_disk`](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_disk) to see if that resource is supported.
 
-Adding support for a resource has two steps:
+Adding support for a resource has four steps:
 
-1. Make a PR for [Magic Modules](https://github.com/GoogleCloudPlatform/magic-modules) to add the necessary code to terraform-google-conversion. Once your PR is merged, the code will be automatically copied into terraform-google-conversion.
-2. Make a PR for terraform-validator that updates the version of terraform-google-conversion and adds tests for the new resource.
+1. Make changes to [Magic Modules](https://github.com/GoogleCloudPlatform/magic-modules) to add any necessary code to terraform-google-conversion.
+2. Make changes to terraform-validator to adds tests for the new resource. Make sure the new tests pass locally.
```

Comment:
Would it be helpful to add something like "Using the generated terraform-google-conversion, make and test your changes in terraform-validator..." ?